сomponents/Nav/Nav.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import classes from "./Nav.module.css";
+
+const Nav = () => {
+  return (
+    <div>
+      <div className={classes.navbar}>
+        <div className="container">

```

Comment:
As I understand you don't have any styles for this class, but you use this. What is the reason?